parsimonious/grammar.py

```diff
@@ -296,64 +299,74 @@ def __init__(self, custom_rules=None):
         """
         self.custom_rules = custom_rules or {}
 
-    def visit_parenthesized(self, parenthesized, (left_paren, _1,
-                                                  expression,
-                                                  right_paren, _2)):
+    def visit_parenthesized(self, node, parenthesized):
         """Treat a parenthesized subexpression as just its contents.
 
         Its position in the tree suffices to maintain its grouping semantics.
 
         """
+        left_paren, _1, expression, right_paren, _2 = parenthesized
```

Comment:
There's no need to number the `_` variables. Let's just call them all `_`, since what they symbolize is that we're throwing away part of the result anyway.